src/backend/joanie/tests/core/test_api_enrollment.py

```diff
@@ -383,7 +425,7 @@ def test_api_enrollment_read_list_filtered_by_was_created_by_order(self):
             user=user, course_run=cr3, was_created_by_order=True
         )
 
-        with self.assertNumQueries(9):
+        with self.assertNumQueries(53):
```

Comment:
ouch! I think we need to optimize it. It should not take so many db queries?

Reply:
In fact this query number is high because it includes thumbnail creation... In real context, once thumbnails have been created the number of query remains normal...

Easy thumbnails offers a signal to generate thumbnail on model save. But I though it's acceptable to generate thumbnails at first request.